src/client/java/daos-java/find_build_paths.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+daos_install_path=$1
+var=$2
+build_vars=../../../../.build_vars.sh
+if [ ! -f "${build_vars}" ]
+then
+  if [ "$var" = "SL_PREFIX" ]
+  then
+    echo "${daos_install_path}"
+  else # SL_PROTOBUFC_PREFIX, default to dev
+    echo "${daos_install_path}/prereq/dev/protobufc/"
+  fi
+else
+  source "${build_vars}"
```

Comment:
One problem is that if protobufc is installed via RPM, I don't think SL_PROTOBUFC_PREFIX gets defined.   I think the ideal thing to do in that scenario would be to skip adding any include paths for protobufc because they would be in system default location.   This is a step in the right direction though so I think it's fine.

Reply:
Yeah.  I'd like to see a run of this PR with `Quick-build:true` in the commit message to see what it does when protobuf-c is installed as an RPM and therefore not built by the DAOS build.  Landing this shouldn't break quick-build.

Reply:
The last commit is with "Quick-build: true". It seems that the RPM build passed.

Reply:
Unfortunately, as @jolivier23 points out, that was a NOOP anyway, since we don't actually build any of this Java stuff in our CI pipeline (yet).

Reply:
I guess it doesn't really matter much.   If protobufc is installed, the compiler will find it and the -L settings will just add non-existent paths which will be ignored.